projects/manage-supervision-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/api/controller/CaseloadController.kt

```diff
@@ -37,4 +51,17 @@ class CaseloadController(private val userService: UserService) {
     @GetMapping("/team/{teamCode}/staff")
     @Operation(summary = "Gets the staff within the team")
     fun getTeamStaff(@PathVariable teamCode: String) = userService.getTeamStaff(teamCode)
+
+    private fun sort(sortString: String): Sort {
+
+        val regex = Regex(pattern = "[A-Za-z0-9]+\\.(ASC|DESC)", options = setOf(RegexOption.IGNORE_CASE))
```

Comment:
nitpick (feel free to ignore) - do you need A-Za-z when the regex is case-insensitive?
```suggestion
        val regex = Regex(pattern = "[A-Z0-9]+\\.(ASC|DESC)", options = setOf(RegexOption.IGNORE_CASE))
```

Reply:
Yep - dont need that. Changed